### PR TITLE
Scrutinizer Auto-Fixes

### PR DESCRIPTION
**Pull request by scrutinizer-auto-fixer**
@DeepDiver1975 requested this pull request.

It consists of patches automatically generated for this project on Scrutinizer:
https://scrutinizer-ci.com/g/owncloud/core/

### CODE REVIEW

File: apps/files_sharing/lib/sharedmount.php [hunk 1]
```diff
@@ -48,6 +48,11 @@ class SharedMount extends MountPoint implements MoveableMount {
 	 */
 	private $user;
 
+	/**
+	 * @param string $storage
+	 * @param string $mountpoint
+	 * @param \OCP\Files\Storage\IStorageFactory $loader
```
Comment (icewind1991) [inline apps/files_sharing/lib/sharedmount.php]: missing `$arguments` ...

Comment (nickvergessen) [inline apps/files_sharing/lib/sharedmount.php]: Fixed

**Comment by nickvergessen** [comment]
Added some more fixes to it

**Comment by MorrisJobke** [comment]
:+1:

**Comment by icewind1991** [comment]
:+1:

**Pull request merged**